ecs_fargate/assets/dashboards/amazon_fargate_overview.json

```
@@ -2,7 +2,7 @@
     "author_info": {
         "author_name": "Datadog"
     }, 
-    "board_title": "ECS-Fargate", 
+    "board_title": "Amazon Fargate", 
```

```suggestion
    "board_title": "Amazon Fargate Overview", 
```

Maybe Amazon Fargate Overview? to be aligned with most dashboard